generatorcore/eb_to_bisko/__init__.py

```diff
@@ -0,0 +1,862 @@
+from dataclasses import dataclass
+
+from generatorcore.inputs import Inputs
+
+from ..utils import div
+from .. import (
+    agri2018,
+    electricity2018,
+    business2018,
+    fuels2018,
+    heat2018,
+    industry2018,
+    lulucf2018,
+    residences2018,
+    transport2018,
+)
+
+
+@dataclass
+class EnergySourceCalcIntermediate:
+
+    eb_energy_from_same_sector: float
+
+    eb_CO2e_cb_from_same_sector: float | None = None
```

Comment:
This probably needs at least one comment somewhere explaining what a field being None means as opposed to a field being 0.  Also why did you need a default value for energy and CO2e_cb at all?